src/sonic-host-services/tests/hostcfgd/hostcfgd_test.py

```diff
@@ -10,7 +10,7 @@
 from .mock_configdb import MockConfigDb
 
 
-swsssdk.ConfigDBConnector = MockConfigDb
+swsscommon.swsscommon.ConfigDBConnector = MockConfigDb
```

Comment:
And also change this line to just "swsscommon.ConfigDBConnector = MockConfigDb"?